### ISSUE TITLE
Provide text and download link for upcoming Yellow Ribbon school year changes 

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#8665]
## Provide text and download link for upcoming Yellow Ribbon school year changes 

Provide text and download link on the Yellow Ribbon tool page for the upcoming school year changes. The business will provide a spreadsheet, ETA sometime in June (early to mid). 

**Example text -- do not use this literally - it is just to capture the meaning/intent:** 
 ‘Need to plan for the new school year? Preview what schools are changing their Yellow Ribbon funding or participation for the upcoming August 2020 to July 2021 school year, blah blah... ‘  

[ ] Provide text that should go on tool page, and recommendation of where on the page. (Content)

[ ] Convert spreadsheet into PDF (Content - with Design and/or Accessibility help?)
- Please check spreadsheet format and column labels, and clean up if needed to match the labels used in our tool. (I sent the business an example spreadsheet they can start with, so they should provide this with the correct FE data labels and just the data columns that are relevant, but always good to check.)

[ ] Validate that PDF is 508 compliant. 

[ ] Host on Github (if we cannot host it in Drupal). 

[ ] Add provided link and text on tool page. (FE engineering) 

[ ] Add analytics tracking to downloads. 
   - We need to track this bc if downloads indicate that students don't want/need this information, we will share this data with the business and propose they stop providing this. 

[ ] Once the 'upcoming' 2020-2021 school information is available from the tool, remove the PDF link. ETA end of July/start of August - switchover in GIDS to be confirmed by business.

**Comment by andaleliz** [comment]
In terms of placement/design, can we use the same solution we had when we were considering historical rates on [the landing page](https://vsateams.invisionapp.com/share/J2X4B3GSRDE#/411695898_Landing_W_Historical_Information) (below the search button), and [on the results page](https://vsateams.invisionapp.com/share/VMX4B4R23NR#/411695897_Results_-_All_Data___Historical_Rates)?  

Maybe on the results page it would need to be a separate `p` element if it's longer than the text we currently have on that screen.

**Comment by kelsonic** [comment]
Any update on this ticket? 🙂

**Comment by AnneHurley** [comment]
@kelsonic  @andaleliz  @stephorkandatx  I will take a pass at this link copy this week.

**Comment by andaleliz** [comment]
Checking in on this - have we received the data from the Yellow Ribbon team? Will a mock-up be helpful once Anne provides the copy updates?

**Comment by DanielleThierryUSDSVA** [comment]
Tagging @bethpottsVADEPO on this since she's taking lead on VBA work from our end. Thanks!

**Comment by bethpottsVADEPO** [comment]
I'm not sure where we are with this work. I'll check in with @AnneHurley and find out the status.

**Comment by AnneHurley** [comment]
> Checking in on this - have we received the data from the Yellow Ribbon team? Will a mock-up be helpful once Anne provides the copy updates?

@bethpottsVADEPO @stephorkandatx  have we gotten the new spreadsheet yet? 
And @andaleliz  Yes I think a mockup would be super helpful.

**Comment by andaleliz** [comment]
Sounds good - I'll make a mock-up once the copy updates are posted.

**Comment by andaleliz** [comment]
I removed myself as an assignee from this issue and created a new one to cover the mock-up as a dependency on this one.   Just lemme know when it's ready to rock :) 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/10278

**Comment by AnneHurley** [comment]
Layout info from Liz: Those are the last two screens in the InVision prototype called Yellow Ribbon MVP v2 https://vsateams.invisionapp.com/share/J2X4B3GSRDE#/screens/411695898_Landing_W_Historical_Information

8:32
Here's the landing page, and results page.  

https://vsateams.invisionapp.com/share/VMX4B4R23NR#/screens/411695897_Results_-_All_Data___Historical_Rates

The solution you see on the landing page here won't work anymore, because Jen wants to move the 3 links in the body copy below the search button. (edited) 
8:33
I think it will work well to put information about upcoming school years above the Search for schools participating in the current academic year by one or all of the terms below line.
8:33
However many sentences it turns out that we need.

**Comment by stephorkandatx** [comment]
@AnneHurley @bethpottsVADEPO @andaleliz @kelsonic we have not received a spreadsheet of this data, yet. Jen has asked for an ETA.

**Comment by AnneHurley** [comment]
@stephorkandatx  Yes I know - thx. I just wanted as much up to date info on where to find the latest mockup/layout of what we'd discussed, esp if Liz is not on the ticket. The above info she just Slacked me so I've captured it for future use.

**Comment by bethpottsVADEPO** [comment]
@jenniferlee-dsva would like to drive these next steps, so I'm adding her and removing @AnneHurley so she can turn to other work.

**Comment by MickinSahni** [comment]
@bethpottsVADEPO to confirm whether these items are complete and if so can close out
